example/webpack.config.js

```diff
@@ -5,20 +5,16 @@ const webpack = require("webpack");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const CopyPlugin = require("copy-webpack-plugin");
 
-const appDirectory = path.resolve(__dirname);
-const { presets, plugins } = require(`${appDirectory}/babel.config.js`);
-// This is only needed in the development repo to transpile the TypeScript files
-const compileNodeModules = ["../../package"].map((moduleName) =>
-  path.resolve(appDirectory, `node_modules/${moduleName}`)
-);
```

Comment:
`const appDirectory = path.resolve(__dirname)` and `__dirname` seem to be the same thing, so we can replace `appDirectory`

```ts
const compileNodeModules = ["../../package"].map((moduleName) =>
  path.resolve(__dirname, `node_modules/${moduleName}`)
);
```

the mapping is creating an array with only this item

```ts
path.resolve(__dirname, "node_modules/../../package")
```

which seems to be the same as

```ts
path.resolve(__dirname, "../package")
```